CMakeLists.txt

```diff
@@ -301,6 +301,7 @@ if (ONEDPL_BACKEND MATCHES "^(tbb|dpcpp|dpcpp_only)$")
         target_compile_definitions(oneDPL INTERFACE
             $<$<OR:$<BOOL:${ONEDPL_USE_DEVICE_FPGA_HW}>,$<BOOL:${ONEDPL_USE_DEVICE_FPGA_EMU}>>:ONEDPL_FPGA_DEVICE>
             $<$<BOOL:${ONEDPL_USE_DEVICE_FPGA_EMU}>:ONEDPL_FPGA_EMULATOR>
+            ONEDPL_USE_DPCPP_BACKEND=1
```

Comment:
Is this necessary?
Or rather - with this change, will our tests still cover the case of "ONEDPL_USE_DPCPP_BACKEND is not defined"?

Reply:
If `-ONEDPL_BACKEND=<dpcpp|dpcpp_only>` option is passed to cmake, it explicitly says that dpcpp backend should be used, what makes it logical to define `ONEDPL_USE_DPCPP_BACKEND` to a non-zero value. 

The case with the macro being undefined should remain covered as well. Let me think how to achieve it.